proto/ignition/msgs/statistic.proto

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "StatisticProtos";
+
+/// \ingroup ignition.msgs
+/// \interface Statistic
+/// \brief A message that contains statistics data.
+
+import "ignition/msgs/header.proto";
+
+message Statistic
+{
+  enum DataType
+  {
+    UNITIALIZED = 0;
```

Comment:
```suggestion
    UNINITIALIZED = 0;
```

Reply:
Fixed in a7ea220481c40b0b273a41d4df9a665276619a01